cmd/lncli/commands.go

```diff
@@ -302,6 +302,11 @@ var sendCoinsCommand = cli.Command{
 				"must satisfy",
 			Value: defaultUtxoMinConf,
 		},
+		cli.BoolFlag{
+			Name: "force, f",
```

Comment:
I wonder if this'll end up breaking any bash scripting automation out there? 

As it's essentially a "breaking" change from the PoV of the CLI...

Reply:
Ah, good catch! Didn't think about that. I did a quick search and at least RaspiBlitz uses the command in a script: https://github.com/rootzoll/raspiblitz/blob/ff6fafdfc9df84a39f23f1191fe82ad0d42400d6/home.admin/BBcashoutWallet.sh#L102

So I decided to throw in an additional check: `if !ctx.Bool("force") && term.IsTerminal(int(os.Stdout.Fd())) {`

That way the confirmation is only displayed if you are on a plain terminal and the standard out is not redirected:

```shell

$ lncli sendcoins --sweepall bcrt1q26avnvcjwutstwjhc0p8dgenyvjwvzrdfv3e50
Amount: 0
Destination address: bcrt1q26avnvcjwutstwjhc0p8dgenyvjwvzrdfv3e50
Confirm payment (yes/no):

$ lncli sendcoins --sweepall bcrt1q26avnvcjwutstwjhc0p8dgenyvjwvzrdfv3e50 | jq -r 'txid'
e5e49332504cc1619117cc5f13d296d9335ecd25ab2ec43d5e635347212fd9e5

```